pkg/ocicni/ocicni.go

```diff
@@ -568,22 +568,22 @@ func (network *cniNetwork) checkNetwork(cacheDir string, podNetwork *PodNetwork,
 func (network *cniNetwork) deleteFromNetwork(cacheDir string, podNetwork *PodNetwork, ifName string, runtimeConfig RuntimeConfig) error {
 	rt, err := buildCNIRuntimeConf(cacheDir, podNetwork, ifName, runtimeConfig)
 	if err != nil {
-		logrus.Errorf("Error deleting network: %v", err)
+		log().Errorf("building CNI runtime config: %v", err)
 		return err
 	}
 
 	netconf, cninet := network.NetworkConfig, network.CNIConfig
-	logrus.Infof("About to del CNI network %s (type=%v)", netconf.Name, netconf.Plugins[0].Network.Type)
+	log().Debugf("deleting CNI network %s (type=%v)", netconf.Name, netconf.Plugins[0].Network.Type)
```

Comment:
Same here; I think this PR should just do the conversion to log() and then we can have a discussion about log levels and verbosity somewhere else.

Reply:
Yep, I will revert it back to `Infof`